spec/controllers/api/v1/headings_controller_spec.rb

```diff
@@ -72,7 +72,7 @@
 
       context 'when record is not present' do
         it 'returns not found if record was not found' do
-          id = heading.goods_nomenclature_item_id.first(4).to_i + 1
+          id = heading.goods_nomenclature_item_id.first(4).next
```

Comment:
This increments 0109 to 0110 instead of 110 which is an invalid heading code, causing this and similar specs to fail when using dynamic item ids - this pattern used elsewhere in this PR